svy/src/helpers/account.ts

```diff
@@ -15,4 +17,62 @@ export function getOrCreateAccount(address: Address): Account {
     account.save();
   }
   return account as Account;
+}
+
+export function receiveSVY(accountAddress: Address, svyReceived: BigInt, block: ethereum.Block): Account {
+  const account = getOrCreateAccount(accountAddress);
+
+  if (account.svyBalance.isZero()) {
+    const protocol = getOrCreateProtocol();
+    protocol.totalSvyHolders += 1;
+    protocol.save();
+  }
+
+  account.svyBalance = account.svyBalance.plus(svyReceived);
+  account.svyBalanceUSD = getSvyBalanceInUSD(account.svyBalance);
+  account.lastUpdatedBN = block.number;
+  account.lastUpdatedTimestamp = block.timestamp;
+  account.save();
+  return account;
+}
+
+export function sendSVY(accountAddress: Address, svySent: BigInt, block: ethereum.Block): Account {
+  const account = getOrCreateAccount(accountAddress);
```

Comment:
you can add `increaseTotalSvyDistributed` here so that every time SVY is sent it updates the source also